shenyu-admin/src/main/java/org/apache/shenyu/admin/exception/ExceptionHandlers.java

```diff
@@ -30,7 +30,7 @@
 
 /**
  * ControllerMethodResolver.
- * https://dzone.com/articles/global-exception-handling-with-controlleradvice.
+ * @see <a href="https://dzone.com/articles/global-exception-handling-with-controlleradvice">global-exception-handling-with-controlleradvice</a>.
```

Comment:
Remove '.' at the end of this line.

Reply:
ok，I fixed it and push the code to git repository